bh/documentation/src/avatar/Avatar.vue

```diff
@@ -26,6 +32,8 @@ import AvatarExample from './examples/AvatarExample';
 import avatarExampleCode from '!!raw-loader!./examples/AvatarExample';
 import AvatarSize from './examples/AvatarSize';
 import avatarSizeCode from '!!raw-loader!./examples/AvatarSize';
+import AvatarExampleImage from './examples/AvatarExampleImage';
```

Comment:
Follow the pattern for examples:

file: examples/ComponentName.vue
component name: ComponentNameExample
component import: ComponentNameExample
component code import: componentNameExampleCode

Reply:
I got it